pandas/tests/frame/test_reductions.py

```diff
@@ -1108,6 +1112,23 @@ def test_any_all_extra(self):
         result = df[["C"]].all(axis=None).item()
         assert result is True
 
+    @pytest.mark.parametrize("axis", [0, 1])
```

Comment:
do we have testing of the skipna=True

Reply:
Some scattered testing elsewhere, added `skipna` parameterization in this test